lib/docker-stdio.js

```diff
@@ -289,6 +332,42 @@ function createDockerFileReadStreamServer(opts, callback) {
 
         zlogin.stdout.pipe(socket);
     }
+
+    function onConnectionZoneNotRunning(socket) {
+        clearTimeout(serverTimeout);
+
+        socket.on('close', function () {
+            tcpServer.close();
+        });
+
+        if (norm.match(new RegExp('^/'))) {
+            norm = norm.slice(1);
+        }
+
+        var chrootTarCmd = __dirname + '/../bin/chroot-gtar';
+        var chrootTarArgs = [
+            '-r', zonepath,
+            '-t', '-',
+            '-m', 'create',
+            '-C', 'root',
+            norm];
+
+        var spawnargs = [chrootTarCmd, chrootTarArgs, { encoding: 'binary' }];
+
+        log.info('docker copy tar spawn args');
+        log.info(JSON.stringify(spawnargs));
+
+        var streamProc = spawn.apply(null, spawnargs);
```

Comment:
Nit: appears like the only reason to use `apply()` is to simplify the logging, I'd prefer complicated-looking logging with explicit params over the indirection of `apply()`.